schema.graphql

```diff
@@ -1,37 +1,44 @@
 # This file is automatically generated by Keystone, do not modify it manually.
 # Modify your Keystone config when you want to change this.
 
-type User {
+type CompanyContactInfo {
   id: ID!
-  name: String
-  email: String
-  password: PasswordState
-  isAdmin: Boolean
+  companyName: String
+  contactName: String
+  contactEmail: String
+  contactPhone: String
+  isPrimaryContact: Boolean
+  country: String
+  city: String
+  address: String
   createdAt: DateTime
-  lastLoginDate: DateTime
-}
-
-type PasswordState {
-  isSet: Boolean!
+  updatedAt: DateTime
+  status: String
 }
 
 scalar DateTime @specifiedBy(url: "https://datatracker.ietf.org/doc/html/rfc3339#section-5.6")
```

Comment:
This isn't even your code, so I am approving anyway, but
maybe it would be better to use the Javascript native method to get the date and time so we aren't dependent on an outside server.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date